### ISSUE TITLE
the issue of noise generation and sample

### ISSUE THREAD

**Issue by wjd23333** [zoubohao/DenoisingDiffusionProbabilityModel-ddpm-#25]
why ues this to generate a noise (saveNoisy = torch.clamp(noisyImage * 0.5 + 0.5, 0, 1))
is this 0.5 necessray?
and also in this function(sampledImgs = sampledImgs * 0.5 + 0.5  # [0 ~ 1])

**Comment by Dystopians** [comment]
This is a normalization step that allows a tensor that would otherwise lie between -1 and 1 to be situated between 0 and 1.
